modules/dream2nix/pip/editable.py

```diff
@@ -29,69 +43,30 @@ def make_editable(
     site_dir,
     editables_dir,
     site_packages,
-    drvs,
     name,
     path,
+    root_dir,
+    store_dir,
 ):
     normalized_name = name.replace("-", "_")
     editable_dir = editables_dir / name
-    drv_out = Path(drvs[name]["out"])
+    full_path = path if path.is_absolute() else root_dir / path
     if editable_dir.exists():
-        print(f"Skipping existing editable source in {editable_dir}", file=sys.stderr)
+        relative_editable_dir = os.path.relpath(editable_dir, root_dir)
```

Comment:
```suggestion
        relative_editable_dir = editable_dir.relative_to(root_dir)
```

lets stick to pathlibs api, as we do use that in the lines above and below